### ISSUE TITLE
Failed to show value:  type OptimizationSolution has no field t

### ISSUE THREAD

**Issue by prbzrg** [SciML/SciMLBase.jl#156]
In Pluto, When I want to see the result of `GalacticOptim` optimization, I get this error:
```julia
Failed to show value:

type OptimizationSolution has no field t

getproperty@optimization_solutions.jl:39[inlined]
rows(::SciMLBase.OptimizationSolution{Float32, 1, Vector{Float32}, SciMLBase.OptimizationProblem{true, SciMLBase.OptimizationFunction{true, GalacticOptim.AutoZygote, ICNF.var"#f#12"{Float32, ICNF.RNODE{Float32}}, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing}, Vector{Float32}, SciMLBase.NullParameters, Nothing, Nothing, Nothing, Nothing, Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}}, Flux.Optimise.AMSGrad, Float32, Nothing})@tabletraits.jl:26
table_data(::SciMLBase.OptimizationSolution{Float32, 1, Vector{Float32}, SciMLBase.OptimizationProblem{true, SciMLBase.OptimizationFunction{true, GalacticOptim.AutoZygote, ICNF.var"#f#12"{Float32, ICNF.RNODE{Float32}}, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing}, Vector{Float32}, SciMLBase.NullParameters, Nothing, Nothing, Nothing, Nothing, Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}}, Flux.Optimise.AMSGrad, Float32, Nothing}, ::IOContext{IOBuffer})@PlutoRunner.jl:1414
show_richest(::IOContext{IOBuffer}, ::Any)@PlutoRunner.jl:1062
var"#sprint_withreturned#60"(::IOContext{Base.DevNull}, ::Int64, ::typeof(Main.PlutoRunner.sprint_withreturned), ::Function, ::SciMLBase.OptimizationSolution{Float32, 1, Vector{Float32}, SciMLBase.OptimizationProblem{true, SciMLBase.OptimizationFunction{true, GalacticOptim.AutoZygote, ICNF.var"#f#12"{Float32, ICNF.RNODE{Float32}}, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing}, Vector{Float32}, SciMLBase.NullParameters, Nothing, Nothing, Nothing, Nothing, Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}}, Flux.Optimise.AMSGrad, Float32, Nothing})@PlutoRunner.jl:1012
format_output_default(::Any, ::Any)@PlutoRunner.jl:920
var"#format_output#48"(::IOContext{Base.DevNull}, ::typeof(Main.PlutoRunner.format_output), ::Any)@PlutoRunner.jl:937
formatted_result_of(::Base.UUID, ::Bool, ::Vector{String}, ::Nothing, ::Module)@PlutoRunner.jl:841
top-level scope@none:1
```

**Comment by ChrisRackauckas** [comment]
@fonsp what is the show method that Pluto calls?

**Comment by fonsp** [comment]
For a `sol::OptimizationSolution`, [Pluto used](https://github.com/fonsp/Pluto.jl/blob/16bba29e0fa488f139698e7ac13fb3d67ec81bd6/src/runner/PlutoRunner.jl#L1485) `Tables.rowaccess(sol)` to see that it is a table-like (which returns `true`), for which we use our own frontend table viewer. 

It [then used](https://github.com/fonsp/Pluto.jl/blob/16bba29e0fa488f139698e7ac13fb3d67ec81bd6/src/runner/PlutoRunner.jl#L1430) `Tables.rows(sol)` to get the data-to-be-displayed, which is implemented in SciMLBase: 
https://github.com/SciML/SciMLBase.jl/blob/a34f5a71604d0d3207fa54f53e0da1dbbb89cd9d/src/tabletraits.jl#L18-L29 

This method accesses `sol.t`, which throws an error for `sol::OptimizationSolution`.

If the table-like trait was not intentional, you could add:
```julia
Tables.isrowtable(::Type{<:AbstractOptimizationSolution}) = false
```

But I am also fixing this on the Pluto side: https://github.com/fonsp/Pluto.jl/pull/2109 It seems that the array/table interface is not reliable to use "in the wild", so we are going to add a `text/plain` fallback, which would have prevented this issue.

**Comment by ChrisRackauckas** [comment]
Oh no, I'd trust it. We were at fault here, since we accidentally applied this to non-timeseries models, which is now fixed in #160.

It would be good for that table view to include the summary text though, so that one knows to interpret it as a table of ODE solution not as a DataFrame.
